src/sonic-config-engine/portconfig.py

```diff
@@ -1,29 +1,48 @@
 #!/usr/bin/env python
 import os
 import sys
+import json
+import ast
+import re
+from collections import OrderedDict
 
+# Global Variable
+LANES = 4
```

Comment:
should not be hardcoded,  the LANES should be the length of the "lanes" in the platform.json.

Reply:
updated as requested 👍